CPP_STYLE_GUIDE.md

```diff
@@ -192,40 +192,35 @@ class FancyContainer {
 
 ### Use `nullptr` instead of `NULL` or `0`
 
-What it says in the title.
+Further reading at [ES.47].
```

Comment:
I’d really like it if we could make these links more expressive instead of just having the item numbers in the text, e.g. `Further reading in the [C++ Core Guidelines][ES.47]`?

Reply:
ack